Makefile

```diff
@@ -29,6 +29,13 @@ KIND_MANAGED_NAMESPACE ?= open-cluster-management-agent-addon
 MANAGED_CLUSTER_NAME ?= managed
 HUB_CLUSTER_NAME ?= hub
 KIND_VERSION ?= latest
+
+# TODO: Remove this since this a workaround until the following is addressed.
+# https://github.com/stolostron/backlog/issues/25698
+ifeq ($(KIND_VERSION), latest)
+	KIND_VERSION = v1.24.4
+endif
```

Comment:
I wish we didn't have to do it like this, but it would be more of a hassle to be "honest" and set the default to v1.24.x, since we would have to update the prow config to require that, instead of latest...

Reply:
Right, and we'd also have to update every single controller to use `v1.24.x` instead of `latest`.